packages/shared/src/styles/base.css

```diff
@@ -232,3 +232,25 @@ img.lazyload:not([src]) {
     opacity: 0;
   }
 }
+
+::-webkit-scrollbar {
+  width: 0.5rem;
+  background-color: transparent;
+}
+
+::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+::-webkit-scrollbar-thumb {
+  border-radius: 0.25rem;
+  background: var(--theme-label-tertiary);
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background: var(--theme-label-primary);
+}
+
+::-webkit-scrollbar-thumb:active {
+  background: var(--theme-label-primary);
+}
```

Comment:
Please add a new line at the end of the file.

Reply:
Yah, I'll add that